app/controllers/games_controller.rb

```diff
@@ -1,4 +1,27 @@
+require 'pry'
 class GamesController < ApplicationController
-  # Add your GamesController code here
+  
+  def index
+    @games = Game.all
+    render json: @games, status: 201 
+  end 
+
+  def show
+    @game = Game.find(params[:id])
+    render json: @game, status: 201    
+  end 
+  
+  def create
+    # binding.pry
+    @game = Game.create(state: params[:state])
```

Comment:
You should always ensure that you are whitelisting your parameters to only allow approved params to be used in the controller.

```ruby
class GamesController < ApplicationController
  #your game routes

  private
    def game_params
      params.permit(state: [])
    end
end
```